test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -83,7 +90,7 @@ UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_peer_failure, dc_mlx5, "dc_mlx5")
 
 
 test_ucp_peer_failure::test_ucp_peer_failure() :
-    m_am_rx_count(0), m_err_count(0), m_err_status(UCS_OK)
+    m_rndv_descs(), m_am_rx_count(0), m_err_count(0), m_err_status(UCS_OK)
```

Comment:
why need 'm_rndv_descs()' ?

Reply:
yes, or release

Reply:
But `receive()` still can return pointer to the request and we will need to store it somewhere and call wait for it later. So it doesn't look like simplification for me at this moment. What are the advantages?